enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,18 +769,30 @@ customizing these parameters.
 For the time being, we may defer the ability to customize the parameters
 described above and address the question in a followup EP for tech preview.
 
-**Resolution**: TBD.
+**Resolution**: Deferred until tech preview.  For dev preview, we do not allow
+customization of the SMCP.  The Ingress Operator simply specifies the
+"gateway-controller" profile.  This profile enables Gateway API for ingress,
+with cluster-wide watches for Gateway API CRs.  It also enables mesh, and
+individual namespaces can be added to this mesh using ServiceMeshMember and
+ServiceMeshMemberRoll CRs.  We will revisit the possibility of allowing the
+cluster admin to configure parameters on the SMCP in tech preview once we have
+some end-user feedback regarding what parameters end-users want to be able to
+customize on the SMCP.
 
 Introducing a new config CRD would lead to another question: _How would we
 expose SMCP parameters without duplicating a significant number of definitions
 from the SMCP CRD?_
 
 The [Service Mesh control plane profiles](https://docs.openshift.com/container-platform/4.12/service_mesh/v2x/ossm-profiles-users.html#ossm-control-plane-profiles_ossm-profiles-users) is potentially useful to resolve
-this last question.  For example, the Ingress Operator could check for a profile
-in a well known location, and the cluster admin could put a profile in that
-location in order to configure the SMCP that the Ingress Operator manages.
-
-**Resolution**: TBD.
+this last question.  For example, the Service Mesh Operator could provide a set
+of profiles covering popular configurations, and the Ingress Operator could
+provide an API for the cluster admin to choose a subset of these profiles in
+order to configure the SMCP that the Ingress Operator manages.
+
+**Resolution**: Deferred until tech preview.  For dev preview, we will hard-code
+the "gateway-controller" profile.  In the future, we may allow the cluster admin
```

Comment:
I don' think we specify the "gateway-controller" profile for dev preview?

Reply:
You're right.  Fixed in https://github.com/openshift/enhancements/compare/5ccb87fd14dd84eebfc38506dc1596a75eeccdfb..d60687328a0112a0472424afaf6ec4cc2611b98c.